lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
```

Comment:
Is this actually used by anything?

Reply:
I'm not sure yet - I think it might be used by `rails scaffold`, but I haven't checked. I lean towards removing it.